2024-03-13 14:02:51.659 [info] 




2024-03-13 14:02:51.673 [info] Extension host agent started.
2024-03-13 14:02:51.790 [info] [127.0.0.1][6a8bd45c][ManagementConnection] New connection established.
2024-03-13 14:02:51.792 [info] [127.0.0.1][de0319a0][ExtensionHostConnection] New connection established.
2024-03-13 14:02:51.930 [info] [127.0.0.1][de0319a0][ExtensionHostConnection] <2031> Launched Extension Host Process.
2024-03-13 14:03:05.007 [info] [127.0.0.1][6a8bd45c][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
2024-03-13 14:03:05.074 [info] [127.0.0.1][de0319a0][ExtensionHostConnection] <2031> Extension Host Process exited with code: 0, signal: null.
2024-03-13 14:03:05.075 [info] Cancelling previous shutdown timeout
2024-03-13 14:03:05.075 [info] Last EH closed, waiting before shutting down
2024-03-13 14:03:11.908 [info] Got delay-shutdown request while in shutdown timeout, delaying
2024-03-13 14:03:11.908 [info] Cancelling previous shutdown timeout
2024-03-13 14:03:11.960 [info] [127.0.0.1][c59b8aa8][ManagementConnection] New connection established.
2024-03-13 14:03:11.996 [info] [127.0.0.1][c3ea4bca][ExtensionHostConnection] New connection established.
2024-03-13 14:03:12.008 [info] [127.0.0.1][c3ea4bca][ExtensionHostConnection] <2185> Launched Extension Host Process.
2024-03-13 14:03:14.322 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:32578)
    at D.updateTitle (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:27710)
    at T.<computed> (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:22925)
    at Object.call (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:34:5061)
    at u.s (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:5165)
    at u.q (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4667)
    at l.value (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4065)
    at m.y (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at m.fire (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process._ (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:25:10025)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-03-13 14:03:36.622 [info] [127.0.0.1][c59b8aa8][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
2024-03-13 14:03:36.669 [info] [127.0.0.1][c3ea4bca][ExtensionHostConnection] <2185> Extension Host Process exited with code: 0, signal: null.
2024-03-13 14:03:36.669 [info] Cancelling previous shutdown timeout
2024-03-13 14:03:36.669 [info] Last EH closed, waiting before shutting down
2024-03-13 14:03:41.998 [info] Got delay-shutdown request while in shutdown timeout, delaying
2024-03-13 14:03:41.999 [info] Cancelling previous shutdown timeout
2024-03-13 14:03:42.060 [info] [127.0.0.1][d4823839][ManagementConnection] New connection established.
2024-03-13 14:03:42.097 [info] [127.0.0.1][6b9b61b7][ExtensionHostConnection] New connection established.
2024-03-13 14:03:42.106 [info] [127.0.0.1][6b9b61b7][ExtensionHostConnection] <2615> Launched Extension Host Process.
2024-03-13 14:03:44.410 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:32578)
    at D.updateTitle (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:27710)
    at T.<computed> (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:22925)
    at Object.call (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:34:5061)
    at u.s (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:5165)
    at u.q (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4667)
    at l.value (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4065)
    at m.y (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at m.fire (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process._ (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:25:10025)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-03-13 14:03:45.234 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:32578)
    at D.updateIcon (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:27763)
    at T.<computed> (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:22925)
    at Object.call (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:34:5061)
    at u.s (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:5165)
    at u.q (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4667)
    at l.value (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4065)
    at m.y (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at m.fire (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process._ (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:25:10025)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-03-13 14:03:45.235 [error] CodeExpectedError: Could not find pty on pty host
    at D.U (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:32578)
    at D.updateTitle (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:27710)
    at T.<computed> (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:43:22925)
    at Object.call (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:34:5061)
    at u.s (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:5165)
    at u.q (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4667)
    at l.value (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:32:4065)
    at m.y (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at m.fire (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process._ (/home/linux/.vscode-server/bin/1e790d77f81672c49be070e04474901747115651/out/vs/platform/terminal/node/ptyHostMain.js:25:10025)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2024-03-13 14:08:42.004 [info] New EH opened, aborting shutdown
2024-03-13 14:20:03.885 [error] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
2024-03-13 14:20:03.887 [error] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
